### DISCUSSION TITLE
监听网段配置

### DISCUSSION THREAD

**Discussion by Cqupt-Psyduck** [AlistGo/alist#4248] [Invalid question]
在data/config.json中有一个属性为"address": "0.0.0.0"表示监听的地址，我将其改为CIDR 地址段如192.168.1.0/24，像实现只接收某一个网段的请求，如家庭内网，但无法启动程序，报错了是怎么回事

**Comment by Rektyfikowany** [comment]
监听地址只能使用ip，网段限制建议通过路由器或反代限制

**Comment by Cqupt-Psyduck** [comment]
这样啊，学艺不精了，哈哈哈，谢谢了哈



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年4月26日(星期三) 晚上9:42
收件人: "alist-org/alist";
抄送: "Author";
主题: Re: [alist-org/alist] 监听网段配置 (Discussion #4248)





 
监听地址只能使用ip，网段限制建议通过路由器或反代限制
 
&mdash;
